src/main/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/jpa/entity/RecommendationEntity.kt

```diff
@@ -100,7 +102,7 @@ data class RecommendationModel(
 ) : Serializable
 
 enum class Status {
-  DRAFT, DOCUMENT_CREATED, DELETED
+  DRAFT, DOCUMENT_CREATED, DELETED, RECALL_CONSIDERED
```

Comment:
Maybe not part of this PR but if `DOCUMENT_CREATED` isn't used in service, should we just remove it?